doc/performance-investigations/0002-use-firestore-in-datastore-mode/summary.md

```diff
@@ -0,0 +1,13 @@
+# Investigate Firestore in Datastore Mode
```

Comment:
The main heading should match the folder name i.e. "Use Firestore in Datastore Mode"